### DISCUSSION TITLE
skytale c plugin

### DISCUSSION THREAD

**Discussion by fiann21** [AlexandreRouma/SDRPlusPlus#469] [Ideas]
maybe you can add this to > SDR# + ScytaleclC = SDR# plugin

link to downloads > https://bitbucket.org/scytalec/scytalec/downloads/

**Comment by the-gobleo-tinkerer** [comment]
As said in #369, SDR# plugins are not supported in SDR++ for both technical and licensing reasons. According to [this RTL-SDR Blog page](https://www.rtl-sdr.com/scytale-c-a-new-inmarsat-std-c-decoder-tekmanoid-std-c-decoder-updates/), Scytale-C is an Inmarsat STD-C decoder, however I have no experience with Inmarsat so I'm not able to help code anything.

**Comment by Frank57** [comment]
I can't even find the exe file of it anywhere, thou in the videos it tells and shows you where they are, when you check it out, there are a lot of files, but no .exe ones, for either Scytale-C or the UI for it? what gives?

**Reply by microp11** [reply]
On Scytalec downloads...

Bitbucket has changed the policy on distribution of binaries. You now need to pay to be able to upload binaries. So they have removed all the links to them. You can still download them, view the repo readme and use the links in there.
<img width="655" height="575" alt="image" src="https://github.com/user-attachments/assets/ac546d8e-aad2-45f3-9706-68955086735a" />
I will move the binaries on bitbucket but until them use above.
